src/lib/queue.js

```diff
@@ -19,7 +19,13 @@ const generateRedisConfig = () => {
       tlsEnabled: true,
       // TLS needs to be set to an empty object for redis on cloud.gov
       // eslint-disable-next-line no-empty-pattern
-      redisOpts: { redis: { password, tls: {} } },
+      redisOpts: {
+        redis: { password, tls: {} },
+        limiter: {
+          max: 1000,
+          duration: 5000,
```

Comment:
We might need a longer max duration if that is 5 sec. Some of the operations we put on the worker is because it is a long task. I think maybe 5 min would be fine.